_lessons/python-scatter/00-introduction.md

```diff
@@ -13,24 +13,38 @@ Learn how to write Python programs that run efficiently on high performance comp
  * understand that there are many ways to write a program (but some ways are better than others)
  * know how to apply different strategies to get your code to run faster
 
+## Getting ready
 
-## Getting started
+The training requires:
 
-The training will be based on the *scatter* code. Clone and switch to the repository:
+ * Python 3 (tested with version 3.6.3)
+ * numpy (1.13.3)
+ * scipy (1.0.0)
+ * setuptools (28.8.0)
+ * Boost library (1.61)
```

Comment:
```suggestion
 * Boost library (1.61)
 * gprof2dot (2017.9.19)
 * Graphviz (tested with 2.30.1)
```

@pletzer - extra dependencies for profiling. Also, most of those versions are probably not required, it should work with other versions too.

Reply:
Thanks @chrisdjscott. Added